build-system-tests/scripts/build-android.sh

```diff
@@ -32,12 +32,12 @@ if [ $BUILD_TOOL == 'expo' ]; then
   npm run android -- -p 19000 >$LOG_FILE &
   npx wait-on -t 20000 tcp:19000
 else
-  log "command" "cd android"
-  cd android
-  log "command" "./gradlew clean" # To prevent "installDebug FAILED" https://stackoverflow.com/a/54955869/12610324
-  ./gradlew clean
-  log "command" "cd .."
-  cd ..
-  log "command" "npm run android"
-  npm run android
+  log "command" "cd android >$LOG_FILE "
+  cd android >$LOG_FILE
+  log "command" "./gradlew clean >$LOG_FILE" # To prevent "installDebug FAILED" https://stackoverflow.com/a/54955869/12610324
+  ./gradlew clean >$LOG_FILE
+  log "command" "cd .. >$LOG_FILE"
+  cd .. >$LOG_FILE
+  log "command" "npm run android >$LOG_FILE"
+  npm run android >$LOG_FILE
```

Comment:
Small nit: Probably not necessary for this PR but to ensure we don't miss this if we update this script can we make a utility function or something that appends the log file to all commands?

Reply:
Yeah I think that makes sense. One of the next action items is to audit all possible failure scenarios so I can address this there, i.e. ensure all the errors get logged and find a cleaner way to make sure we're logging everything.